File: EjerciciosIA/sumaIA.js
//Página HTML + JavaScript que suma dos números
<!DOCTYPE html>
<html>
<head><title>Suma JS</title></head>
<body>
    <input type="number" id="num1" placeholder="Número 1">
    <input type="number" id="num2" placeholder="Número 2">
    <button onclick="sumar()">Sumar</button>
    <p id="resultado"></p>

    <script>
        function sumar() {
            const a = parseFloat(document.getElementById('num1').value);
            const b = parseFloat(document.getElementById('num2').value);
            const resultado = a + b;
            document.getElementById('resultado').innerText = "Resultado: " + resultado;
        }
    </script>
</body>
</html>

File: EjerciciosIA/sumaIA.java
//Programa en Java que toma dos números y devuelve la suma
import java.util.Scanner;

public class SumaNumeros {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Introduce el primer número: ");
        int num1 = sc.nextInt();
        System.out.print("Introduce el segundo número: ");
        int num2 = sc.nextInt();
        int suma = num1 + num2;
        System.out.println("La suma es: " + suma);
        sc.close();
    }
}
